src/public/js/rs.js

```diff
@@ -1,16 +1,31 @@
 'use strict';
 var RS = (function() {
-  // (actually ROT-13 right now, as a dummy example. We can replace w/ RS)
-  var ROT = 13;
+  // Using a RS with 256 symbol size 255 symobls in a codeword and 245 message symbols.
+  // n-k = 10 code symbols. Can correct up to 5 symbol errors in each 255 symbol block
```

Comment:
I feel like these numbers shouldn't be in the documentation because all of the functions are parameterized & we're letting n and k be settable